File: Sorting Algorithms/Bubble Sort.cpp
#include <iostream>

using namespace std;

void bubbleSort( int A[], int N );

int main()
{
    int N;

    cout << "Enter No of Elements in array : ";
    cin >> N;

    int arr[N];

    cout << "Enter Elements of array : ";
    for ( int  i = 0; i < N; i++ )
    {
        cin >> arr[i];
    }

    bubbleSort( arr, N );

    cout << "Elements in Sorted order are " << endl;
    for ( int i = 0; i < N; ++i )
    {
        cout << arr[i] << " ";
    }

    return 0;
}

void bubbleSort( int A[], int N )
{
    for ( int i = 1; i < N; i++ )
    {
        for ( int j = 0; j < (N - i) ; j++ )
        {
            if ( A[j] > A[j+1] )
            {
                int temp = A[j];
                A[j] = A[j+1];
                A[j+1] = temp;
            }
        }
    }
}

File: Sorting Algorithms/Python/Bubble Sort.py
def bubbleSort( arr, len ) :
    for i in range(len - 1):
        for j in range(len - 1 - i):
            if arr[j] > arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]


len = int( input( "Enter Length of array : " ) )

arr = []

for i in range( len ) :
    x = int( input( "Enter value in array : " ) )
    arr.append( x )

print( "Unsorted array is : ", arr )

bubbleSort( arr, len )

print( "Elements in sorted order are : ", arr )